Tests/API/RPC Fixtures/transaction.json

```diff
@@ -12,7 +12,8 @@
                         "block_hash": "0xca97a033aa9765914bc0b7887ff8723037614df11d06783555ff97e9a0e72b55",
                         "cell": {
                             "index": "0",
-                            "tx_hash": "0x602b713fac38c0224bc2e8ba0ec9c8363d7d1e508897b33c32f45d7554883c81"
+                            "tx_hash": "0x602b713fac38c0224bc2e8ba0ec9c8363d7d1e508897b33c32f45d7554883c81",
+                            "hash_type": "Data"
```

Comment:
Seems the docs from CKB there is no "hash_type" placed here?

Reply:
> Seems the docs from CKB there is no "hash_type" placed here?

There is. See https://github.com/nervosnetwork/ckb/tree/develop/rpc#examples-11.

Reply:
Fixed in afa017a.